x/tokenomics/token_logic_module/reimbursement_requests.go

```diff
@@ -0,0 +1,132 @@
+package token_logic_module
+
+import (
+	"context"
+	"fmt"
+
+	cosmoslog "cosmossdk.io/log"
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	servicetypes "github.com/pokt-network/poktroll/x/service/types"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	tokenomicstypes "github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+var _ TokenLogicModule = (*tlmGlobalMintReimbursementRequest)(nil)
+
+type tlmGlobalMintReimbursementRequest struct {
+	daoRewardBech32 string
```

Comment:
We use `Addr` everywhere whenever we decode one. I believe (but we need to double check) that `___Addr` does refer to a `Bech32` address.

1. Can you double check what we do elsewhere
2. Can you make the suffix Addr if we need to be consistent?
3.